### ISSUE TITLE
CallShoebilFunction crashes when Java method returns a boolean

### ISSUE THREAD

**Issue by Justas-S** [Shoebill/ShoebillPlugin#1]
Here's a tested example. 

```
private boolean isPlayerLoggedIn(int id) {
        LtrpPlayer player = LtrpPlayer.get(id); // This class is very similiar to Player
        if(player != null) {
            return player.isLoggedIn();
        }
        return false;
    }
```

```
        managerNode.registerHandler(AmxLoadEvent.class, e-> {
                    e.getAmxInstance().registerFunction("isPlayerLoggedIn", objects -> {
                                objects[1] = isPlayerLoggedIn((Integer) objects[0]);
                        return 1;
                    }, Integer.class, Integer.class);
        });
```

Pawn code:

```
stock IsPlayerLoggedIn(playerid)
{
    new ishe; // Also tried adding bool tag here
    CallShoebillFunction("isPlayerLoggedIn", playerid, ishe);
    return ishe;
}
```

And that makes `CallShoebillFunction` crash. Here's the AMX backtrace in case its needed:

```
[14:58:23] [debug] AMX backtrace:
[14:58:23] [debug] #0 native CallShoebillFunction () from Shoebill.DLL
[14:58:23] [debug] #1 00202b5c in IsPlayerLoggedIn (playerid=2) at F:\AAAA\LTRP\gamemodes\ltrp.pwn:26893
[14:58:23] [debug] #2 001e1cd4 in public Sekunde () at F:\AAAA\LTRP\gamemodes\ltrp.pwn:23884
[14:58:23] [debug] Native backtrace:
[14:58:23] [debug] #0 540cdfd6 in ?? () from C:\Program Files (x86)\Java\jdk1.8.0_05\jre\bin\server\jvm.dll
[14:58:23] [debug] #1 5412ad91 in ?? () from C:\Program Files (x86)\Java\jdk1.8.0_05\jre\bin\server\jvm.dll
[14:58:23] [debug] #2 5412cc1c in ?? () from C:\Program Files (x86)\Java\jdk1.8.0_05\jre\bin\server\jvm.dll
[14:58:23] [debug] #3 552530bb in ?? () from plugins\Shoebill.DLL
[14:58:23] [debug] #4 5525f2f8 in ?? () from plugins\Shoebill.DLL
[14:58:23] [debug] #5 004010b6 in ?? () from samp-server.exe
[14:58:23] [debug] #6 554262ca in ?? () from plugins\crashdetect.DLL
[14:58:23] [debug] #7 55428b28 in ?? () from plugins\crashdetect.DLL
[14:58:23] [debug] #8 554209c7 in ?? () from plugins\crashdetect.DLL
[14:58:23] [debug] #9 5542631a in ?? () from plugins\crashdetect.DLL
[14:58:23] [debug] #10 55287e7f in ?? () from plugins\Shoebill.DLL
[14:58:23] [debug] #11 004829f8 in ?? () from samp-server.exe
[14:58:23] [debug] #12 0049124a in ?? () from samp-server.exe
[14:58:23] [debug] #13 0048d46b in ?? () from samp-server.exe
[14:58:23] [debug] #14 0049b431 in ?? () from samp-server.exe
[14:58:23] [debug] #15 0049b441 in ?? () from samp-server.exe
```

**Comment by 123marvin123** [comment]
Hi,

thanks for opening an issue. I will try it myself, and fix it, but for now please use Integer and cast it to an bool until I fixed this problem.

**Comment by 123marvin123** [comment]
Fixed in Shoebill 1.2
